projects/cloud/app/graphql/types/role_type.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module Types
+  class RoleType < Types::BaseEnum
+    value "admin"
```

Comment:
Correct me but, this then means we have 3 roles: admin, user, and owner of the account?

Reply:
Yes, I'm for adding `owner` in a different PR, though. There are some intricacies tied to changing it and right now, `admin` == `owner` in terms of what their capabilities.